tests/TestHarness/Cluster.py

```diff
@@ -244,7 +244,7 @@ def launch(self, pnodes=1, unstartedNodes=0, totalNodes=1, prodCount=21, topo="m
             time.sleep(2)
         loggingLevelDictString = json.dumps(self.loggingLevelDict, separators=(',', ':'))
         args=(f'-p {pnodes} -n {totalNodes} -d {delay} '
-              f'-i {datetime.datetime.now(datetime.UTC).strftime("%Y-%m-%dT%H:%M:%S.%f")[:-3]} -f {producerFlag} '
```

Comment:
This change isn't happy on python 3.12    Sorry I missed this with my earlier patch.   We can merge this and I'll submit a follow-up fix that works for both.

Reply:
Why is python 3.12 having issues? Is it the datetime.datetime?  I saw that utcnow is in the code, so I didn't think it would be a problem.